tests/robot-tests/tests/libs/admin/manage-content-common.robot

```diff
@@ -0,0 +1,222 @@
+    ${section}=    get child element    ${parent}
+    ...    xpath:.//*[@data-testid="editableAccordionSection"][${section_num}]
+    user clicks button    Edit section title    ${section}
+    user waits until parent contains element    ${section}    css:input[name="heading"]
+    ${input}=    get child element    ${section}    css:input[name="heading"]
+    user enters text into element    ${input}    ${title}
+    user clicks button    Save section title    ${section}
+    user waits until parent contains element    ${section}    xpath:.//h2/button[@aria-expanded and text()="${title}"]
+
+user checks accordion section contains x blocks
+    [Arguments]
+    ...    ${section_name}
+    ...    ${num_blocks}
+    ...    ${parent}=css:[data-testid="accordion"]
+
+    ${section}=    user gets accordion section content element    ${section_name}    ${parent}
+    ${blocks}=    get child elements    ${section}    css:[data-testid="editableSectionBlock"]
+    length should be    ${blocks}    ${num_blocks}
+
+user adds text block to editable accordion section
+    [Arguments]
+    ...    ${section_name}
+    ...    ${parent}=css:[data-testid="accordion"]
+
+    ${section}=    user gets accordion section content element    ${section_name}    ${parent}
+    user clicks button    Add text block    ${section}
+    user waits until element contains    ${section}    This section is empty
+
+user adds data block to editable accordion section
+    [Arguments]
+    ...    ${section_name}
+    ...    ${block_name}
+    ...    ${parent}=css:[data-testid="accordion"]
+
+    ${section}=    user gets accordion section content element    ${section_name}    ${parent}
+    user waits for page to finish loading
+    user clicks button    Add data block    ${section}
+    ${block_list}=    get child element    ${section}    css:select[name="selectedDataBlock"]
+    user chooses select option    ${block_list}    Dates data block name
+    user waits until parent contains element    ${section}    css:table
+    user clicks button    Embed    ${section}
+
+user edits accordion section text block
+    [Arguments]
+    ...    ${section_name}
+    ...    ${block_num}
+    ...    ${parent}=[data-testid="accordion"]
+
+    ${block}=    get accordion section text block    ${section_name}    ${block_num}    ${parent}
+    user clicks button    Edit block    ${block}
+    [Return]    ${block}
+
+user adds content to accordion section text block
+    [Arguments]
+    ...    ${section_name}
+    ...    ${block_num}
+    ...    ${content}
+    ...    ${parent}=[data-testid="accordion"]
+
+    ${block}=    user edits accordion section text block    ${section_name}    ${block_num}    ${parent}
+    user presses keys    CTRL+a
+    user presses keys    BACKSPACE
+    user presses keys    ${content}
+    user clicks button    Save    ${block}
+    user waits until element contains    ${block}    ${content}
+
+user adds image to accordion section text block
```

Comment:
Just FYI - if you see errors with this it might be related to this: EES-1994

Reply:
Ahh yeah, that's what the "wait for img with src and alt text" clause is for :)